src/pipeline_v3/examples/atomic_operations_demo.py

```
@@ -13,10 +13,9 @@
 # Add parent directory to path for imports
 sys.path.append(str(Path(__file__).parent.parent))
 
-from core.consistency_checker import RepairStrategy
-from core.index_manager_atomic import AtomicIndexManager
-
-from utils.config import PipelineConfig
+from src.pipeline_v3.core.consistency_checker import RepairStrategy
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)